handlers/friday.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"time"
+
+	"io"
+	"net/http"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var (
+	fridayTrigger = "sextou"
+	fridayGifUrl  = "https://media.tenor.com/zGlEbV_bTnIAAAAC/kowalski-familia.gif"
+)
+
+func FridayHandler(s *discordgo.Session, m *discordgo.MessageCreate) {
+	if !strings.Contains(m.Content, fridayTrigger) {
+		return
+	}
+
+	message := &discordgo.MessageSend{
+		Files: []*discordgo.File{},
+	}
+
+	switch time.Now().Weekday() {
+		case time.Friday:
+			message.Content = "Sextouu família"
```

Comment:
The same here, maybe you could have a list of possible messages and get one of them randomly.

Reply:
Nice, i generated a random gif for all the days with a different search for the Thursday